IFC4x3/Properties/s/Smoke_24x_PWjRXByRIpwhg2SIFm/Documentation.md

```diff
@@ -0,0 +1 @@
+Maximum level of smoke that the equipment complies with
```

Comment:
Same as above "level of smoke" is ambiguous.
```suggestion
Maximum level of smoke that the equipment complies with.
```

Reply:
can we have the measurement unit please!